src/worker-helpers.js

```diff
@@ -160,14 +160,17 @@ export function getCLIEngineOptions(type, config, rules, filePath, fileDir, give
     cliEngineConfig.ignorePath = ignoreFile
   }
 
-  if (config.eslintRulesDir) {
-    let rulesDir = resolveEnv(config.eslintRulesDir)
-    if (!Path.isAbsolute(rulesDir)) {
-      rulesDir = findCached(fileDir, rulesDir)
-    }
-    if (rulesDir) {
-      cliEngineConfig.rulePaths = [rulesDir]
-    }
+  if (config.eslintRulesDirs.length > 0) {
+    cliEngineConfig.rulePaths = []
+    config.eslintRulesDirs.forEach((path) => {
+      let rulesDir = resolveEnv(path)
+      if (!Path.isAbsolute(rulesDir)) {
+        rulesDir = findCached(fileDir, rulesDir)
+      }
+      if (rulesDir) {
+        cliEngineConfig.rulePaths.push(rulesDir)
+      }
```

Comment:
__*Preference*__

I think this could be a bit cleaner by using a more functional approach:

```js
function resolveRulesDir(path) {
    const rulesDir = cleanPath(path)
    if (!Path.isAbsolute(rulesDir)) {
        return findCached(fileDir, rulesDir)
    }
    return rulesDir
}

cliEngineConfig.rulePaths = config.eslintRulesDirs
    .map(resolveRulesDir)
    .filter(path => path)
```

Reply:
I sidestepped the whole issue of coming up with a name and just left it in the `map` 😉.